docs/textual_definitions_SOP.md

```diff
@@ -0,0 +1,45 @@
+2. Similarly, a prefectly accurate minimal definition of a type II pneuomocyte is an epithelial cell that has an 'alveolar lamellar body' (a unique structure only found in these cell types). But this is useless information to a use who knows nothing about this structure (probably most biologists) or who has to annotated data that does not resolve this structure.
+
+A second use of ontologies is encode to knowledge in the form of useful formal links between ontology terms.  For example, in CL we record function and cell components via links to gene ontology terms, location via links to CL and lineage via links to other CL terms. Not all of this information is particularly useful for recognising a cell type, but it is of use to our users and so we often record it in CL using formal relationships. This is relevant to ontology definitions because it is good practise for formal and textual definitions to match, and textual definitions are the place we encode supporting references.
+
+We can't of course, include every known piece of information about a cell in a definition (e.g. all genes expressed). However extended information is useful to our users - especially where it includes information relevant to human physiology and disease. To support this, we propose a second, extended description field which can contain information about minor (secondary) functions and disease.  It can also contain information about properties that may not apply to all subclasses - this is especially useful for t-types in the brain where there is typically sparse knowledge of the extent of variation in morphology and function under each t-type.  In these cases, information should be included about where these proporites do apply.
+
+![image](https://github.com/obophenotype/cell-ontology/assets/112839/eeb45ba5-96c7-4a3e-b68c-62530c043798)
+
+
+### SOP
+
+Text in the definition field should be no longer than one short paragraph and should follow a classic genus, diffentia and gloss type structure:
+   - genus: what type of cell is it (e.g. epithlial cell)
+   - differentia: a list of properties that can be used to distinguish it from other similar cell types, especially those in the same tissue/organ context.  _This should include location_ unless the term is so abstract that this is not possible. We should be liberal in listing properties here in order to support multiple communities who will have different types of data and understanding. Structural and functional properties are preferred over molecular markers unless cell types are named for these markers (we currently make an exception for this for the immune system where classical definitions are tightly linked to cell surface markers).
```

Comment:
```suggestion
   - differentia: a list of properties that can be used to distinguish it from other similar cell types, especially those in the same tissue/organ context and those of the same genus.  _This should include location_ unless the term is so abstract that this is not possible. We should be liberal in listing properties here in order to support multiple communities who will have different types of data and understanding. Structural and functional properties are preferred over molecular markers unless cell types are named for these markers (we currently make an exception for this for the immune system where classical definitions are tightly linked to cell surface markers).
```

Reply:
I think the naming rule expressed here is not quite right. There could be/might be names that do contain reference to markers but the cell is characterized via markers as ground-truth. (But the point is broadly correct as a rule-of-thumb)

Reply:
I agree - I think we haven't quite got the balance right in the advice on markers yet.  

How about:

"Structural and functional properties are preferred over molecular markers unless cell types are named for these markers or they are generally accepted as definitional by the community.  Care should be taken not to attach species-specific markers to species-general cell types.  Please note that while CL supports recording multiple marker sets for cell types, ideally with provenance, evidence and confidence, this is supported is outside of the core definition text."